content/docs/notebooks/setup.md

```diff
@@ -22,7 +22,7 @@ Summary of steps:
 1. Follow the [Kubeflow getting-started guide](/docs/started/getting-started/) 
   to set up your Kubeflow deployment and open the Kubeflow UI.
 
-1. Click **Notebook Servers** in the left-hand panel of the Kubeflow UI.
+1. Click **Notebooks** in the left-hand panel of the Kubeflow UI.
```

Comment:
Is this the latest UI? It seems to have reverted to the Kubeflow v0.5 UI. I haven't yet installed Kubeflow v0.6.2, but in Kubeflow v0.6.1 we removed the TFJob UI, added the Artifact store, and made other changes. These changes seem to have been reverted in your screenshot. See the [v0.6.1 screenshot](https://www.kubeflow.org/docs/images/jupyterlink.png) compared with [your screenshot](https://deploy-preview-1134--competent-brattain-de2d6d.netlify.com/docs/images/jupyterlink.png) - your screenshot says **build version v0.5.0** at the bottom.

Reply:
> ? It see

Oh, I see. my kubeflow in this env is v0.5.*, and this UI is not the latest one.
Thanks a lot for your info . 
I will close this pr.